src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3955,4 +3955,10 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="DynamoXmlFileFormat" xml:space="preserve">
     <value>Dynamo 1.x file format</value>
   </data>
-</root>
+  <data name="MessagePackageOlderDynamo" xml:space="preserve">
+    <value>The package or one of its dependencies use an older version of {0} than you are currently using. Do you want to continue?</value>
```

Comment:
Do we want to add to the message that the package may not work properly due to this reason?

Reply:
I think I will take Tibi's suggestion here if there are no objections:
1. keep the message simple.
2. link to extended documentation in the dev primer or docs browser.